task/01-strings-tasks.js

```diff
@@ -68,8 +68,8 @@ function getStringFromTemplate(firstName, lastName) {
  *   'Hello, John Doe!' => 'John Doe'
  *   'Hello, Chuck Norris!' => 'Chuck Norris'
  */
-function extractNameFromTemplate(value) {
-    throw new Error('Not implemented');
+function extractNameFromTemplate(value) {    
+    return value.substring(7,value.length-1);
 }
```

Comment:
looks good, but please note substring can work with negative numbers also to take symbols from end. In the case `substring(7, -1);` can do the same.

Reply:
fixed

Reply:
done